analyze.info.yml

```diff
@@ -1,7 +1,6 @@
 name: 'Analyze'
 type: module
 description: 'API for adding information in the Analyze tab of entities'
-core_version_requirement: ^10
-package: Custom
-dependencies:
-  - drupal:node
```

Comment:
This module should declare views as dependency because it installs a view

```
  Configuration objects provided by <em class="placeholder">analyze_google_analytics</em> have unmet dependencies: <em class="placeholder">v  
  iews.view.analyze_google_analytics (views)</em> 
```